notifier/victorops-notifier.go

```diff
@@ -0,0 +1,120 @@
+package notifier
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	log "github.com/AcalephStorage/consul-alerts/Godeps/_workspace/src/github.com/Sirupsen/logrus"
+)
+
+type VictorOpsNotifier struct {
+	NotifName  string
+	APIKey     string
+	RoutingKey string
+}
+
+type VictorOpsEvent struct {
+	// Explicitly listed by http://victorops.force.com/knowledgebase/articles/Integration/Alert-Ingestion-API-Documentation/
+	MessageType       string `json:"message_type"`
+	EntityId          string `json:"entity_id"`
+	Timestamp         uint32 `json:"timestamp"`
+	StateMessage      string `json:"state_message"`
+	MonitoringTool    string `json:"monitoring_tool"`
+	EntityDisplayName string `json:"entity_display_name"`
```

Comment:
Pretty sure these are all the default serializer tag translations?

but having them explicit doesn't hurt.